sdk/samples/iot/templates/iothub/test-resources-post.ps1

```diff
@@ -92,9 +102,11 @@ if ($LASTEXITCODE -ne 0)
   exit $LASTEXITCODE
 }
 
+$hub_obj = waitForActiveHub
```

Comment:
nit: Should we add a custom "Write-Host" line above each call to `waitForActiveHub`, so we can differentiate which step we happen to be waiting on for next?
i.e.:
Downloading root cert
Waiting for active hub
Adding SAS device